shop-info-service/handlers/createProduct.ts

```diff
@@ -0,0 +1,32 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import 'source-map-support/register';
+import { Client } from 'pg';
+
+import getSampleResponse from '../helpers/getSampleResponse';
+import getErrorResponse from "../helpers/getErrorResponse";
+import dbOptions from "../helpers/getDbOptions";
+
+
+export const createProduct: APIGatewayProxyHandler = async (event) => {
+  const {title, description, price, count} = JSON.parse(event.body);
+  const client = new Client(dbOptions);
+  await client.connect();
```

Comment:
If connect to database fail, lambda will return 500 "Internal server error", you sure that this is expected behaviour?

Reply:
Yes, I forgot fix it